.github/workflows/deploy-docs.yml

```diff
@@ -17,9 +17,13 @@ jobs:
             - name : Checkout
               uses: actions/checkout@v4
 
-            - name: Install Doxygen
-              run: sudo apt-get install doxygen graphviz -y
-              shell: bash
+            - name: Install Custom Doxygen Version 
+              uses: AdarshRawat1/Install-Doxygen@v1.0
```

Comment:
This is your own action? Neat. However, for maintenance reasons it might be easier to just inline this particular installation.

Reply:
Sure, I'll do that !!

However this action is responsible only for installating particular version of doxygen being passed, and it would be cleaner this way.

That is the reason I thought it would be better to have a custom action.

Reply:
As there isn't much that we'll have to maintain here other than the version.

Reply:
The main reason for inlining is that we maintain the installation directly here in P4C whereas with the other approach we have to depend on an external action we do not necessarily own.

Reply:
Sure thing !